online.owncast.Owncast.json

```diff
@@ -0,0 +1,21 @@
+{
+	"app-id": "online.owncast.Owncast",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "23.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "/app/owncast",
+	"modules": [
+		{
+			"name": "Owncast",
+			"buildsystem": "simple",
+			"build-commands": ["cp ./owncast /app/"],
+			"sources": [
+				{
+					"type": "file",
+					"path": "/build/flatpak/owncast"
+				}
+			]
+		}
+	],
+	"finish-args": ["--socket=x11", "--share=network"]
```

Comment:
When installing this it required X11 permissions, even though this is server software and not going to necessarily be installed on a desktop machine. Is X11 permissions required?

![image](https://github.com/owncast/owncast/assets/414923/eeaf8f45-c2af-4e4a-af14-4f06dd6a4360)